docs/symptom-survey/contingency-tables.md

```diff
@@ -91,6 +99,11 @@ is a concatenated list of other grouping variables used, ordered
 alphabetically. Values for variables used in file naming align with those
 within files as specified in the column section below.
 
+"Rollup" files containing all time periods for a given period type-aggregation
+type combination have names of the form:
+
+    {period_type}_{geo_type}_{aggregation_type}.csv.gz
+
```

Comment:
I think if you move this description up a bit (before the naming of the non-rollup files), this is ready to merge